main_routers/workshop_router.py

```
@@ -0,0 +1,1420 @@
+            "folder_path": folder_path,  # 返回绝对路径
+            "default_path_used": default_path_used
+        })
+        
+    except Exception as e:
+        logger.error(f"扫描本地创意工坊物品失败: {e}")
+        return JSONResponse(content={"success": False, "error": str(e)}, status_code=500)
+
+# 获取创意工坊配置
+
+@router.get('/local-items/{item_id}')
+async def get_local_workshop_item(item_id: str, folder_path: str = None):
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)